java/java.api.common/src/org/netbeans/modules/java/api/common/singlesourcefile/DebugProcess.java

```diff
@@ -23,25 +23,44 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.concurrent.Callable;
 import java.util.logging.Level;
-import java.util.logging.Logger;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import org.netbeans.api.java.platform.JavaPlatformManager;
 import org.openide.filesystems.FileObject;
 import org.openide.filesystems.FileUtil;
 
-/**
- *
- * @author Arunava Sinha
- */
-class DebugProcess {
+final class DebugProcess implements Callable<Process> {
```

Comment:
Nitpick: `LaunchProcess` could be more appropriate as it does debug / run.

Reply:
done in  4f9aedf